backend/test_observer/data_access/models_enums.py

```diff
@@ -35,6 +35,15 @@ class TestExecutionStatus(Enum):
     NOT_TESTED = "NOT_TESTED"
 
 
+class TestExecutionReviewStatus(str, Enum):
+    UNDECIDED = "UNDECIDED"
+    MARKED_AS_FAILED = "MARKED_AS_FAILED"
+    APPROVED_INCONSISTENT_TEST_DEF = "APPROVED_INCONSISTENT_TEST_DEF"
+    APPROVED_UNSTABLE_PHSYICAL_INFRA = "APPROVED_UNSTABLE_PHSYICAL_INFRA"
+    APPROVED_FAULTY_HARDWARE = "APPROVED_FAULTY_HARDWARE"
+    APPROVED_GENERIC = "APPROVED_GENERIC"
```

Comment:
Should we perhaps have instead of `APPROVED_GENERIC` have both `APPROVED_AUTOMATIC` and `APPROVED_OTHER`?